src/Components/Components/src/ComponentBase.cs

```diff
@@ -255,9 +255,9 @@ private async Task CallStateHasChangedOnAsyncCompletion(Task task)
             StateHasChanged();
         }
 
-        Task IHandleEvent.HandleEventAsync(EventHandlerInvoker binding, UIEventArgs args)
+        Task IHandleAfterEvent.HandleEventAsync(EventCallbackWorkItem callback, object arg)
         {
-            var task = binding.Invoke(args);
+            var task = callback.InvokeAsync(arg);
```

Comment:
Since it actually does invoke the callback itself (instead of just doing stuff *after* the callback was invoked), would the original name `IHandleEvent` actually be a better description?

Reply:
Sure, but this will be my last iteration of renaming this thing until we do an all-up api review.